src/azure-cli/azure/cli/command_modules/role/_help.py

```diff
@@ -373,7 +373,7 @@
     short-summary: Certificate to use for credentials.
     long-summary: When used with `--keyvault,` indicates the name of the cert to use or create. Otherwise, supply a PEM or DER formatted public certificate string. Use `@{path}` to load from a file. Do not include private key info.
   - name: --create-cert
-    short-summary: Create a self-signed certificate to use for the credential.
+    short-summary: Create a self-signed certificate to use for the credential. Only current system user has read/write permission to this certificate.
```

Comment:
Why is "system" mentioned here? This may be confusing...

```suggestion
    short-summary: Create a self-signed certificate to use for the credential. Only current the user has read/write permission to this certificate.
```

Reply:
I want to clarify that it is current OS user, not azure user. Do you have a better suggestion? Do you mean `Only current user` ?

Reply:
Good point, how about 

> Only the current OS user (not Azure user) has read/write permission to this certificate.